scripts/package.json

```diff
@@ -50,7 +50,7 @@
   "bundlesize": [
     {
       "path": "../apps/test-bundle-button/dist/test-bundle-button.min.js",
-      "maxSize": "47.6 kB"
```

Comment:
Worth noting that the current size in the most up to date version of master is 49.6 so this is not a bump of 2KB